scripts/data.js

```diff
@@ -7,7 +7,17 @@ module.exports = {
     demoURL: "https://gatsbycontentfulhomepage.gatsbyjs.io/",
     deploynow:
       "https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/gatsbyjs/gatsby-starter-contentful-homepage",
+    isTypescript: false,
   },
+  // "contentful-ts": {
+  //   repo: "",
+  //   cms: "Contentful",
+  //   name: "contentful",
+  //   dirname: "contentful-plugin",
+  //   demoURL: "",
+  //   deploynow: "",
+  //   isTypescript: true,
+  // },
```

Comment:
I will either remove these commented out items or fill in the details depending on whether we decide to keep this one repo as our single source, or decide it's better to split the TS version off into it's own source repository.

Reply:
Duly noted. That is my desire as well, however there's two issues to consider:

1. Currently when transpiling to `esnext` all new lines are stripped, so the resulting "source" code doesn't have spacing in it that you'd expect from "source" code. From what I've read, this should be able to be mitigated with some further processing (e.g., replacing empty lines with a code comment, transpiling, and then replacing that code comment with an empty line on the other side).
2. Fielding PRs from the community would require people to write in TS. This was something I believe we were trying to avoid to not discourage contributions from JS folks.

Reply:
Just an update on this front — was able to resolve the issue I described in point 1 and with that feel confident moving forward with the plan to have a single source in TS.